inc/class-provider.php

```diff
@@ -35,6 +35,55 @@ public function __construct( Factory $factory ) {
 		$this->factory = $factory;
 	}
 
+	/**
+	 * Return the provider ID.
+	 *
+	 * @return string
+	 */
+	public function get_id(): string {
+
+		return 'wordpress'; // phpcs:ignore
```

Comment:
Meh, this snff is annoying. I suggest to remove it from the coding standards, actually.
And if we don't want to do this, let's definitely not turn off all of PHPCS for this line, but be specific about the sniff.